CorsixTH/Lua/dialogs/resizables/hotkey_assign.lua

```diff
@@ -487,21 +487,23 @@ function UIHotkeyAssign:UIHotkeyAssign(ui, mode)
 
   self:showKeyPane("global")
 
-  -- "Accept" button
+  -- "Cancel" button (rename to "Back" and move to left)
```

Comment:
You can just update the comment to `"Back" button` here. It'll all be understood 🙂